forge/test/models/pytorch/vision/regnet/test_regnet.py

```diff
@@ -0,0 +1,54 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
+
+# SPDX-License-Identifier: Apache-2.0
+import pytest
+from transformers import AutoImageProcessor, RegNetModel, RegNetForImageClassification
+import forge
+from forge.verify.verify import verify
+from .utils.image_utils import preprocess_input_data
+
+
+@pytest.mark.nightly
+@pytest.mark.model_analysis
+@pytest.mark.xfail(
+    reason="RuntimeError: TT_FATAL @ tt-forge-fe/third_party/tt-mlir/third_party/tt-metal/src/tt-metal/ttnn/cpp/ttnn/tensor/tensor_utils.cpp:474: new_volume == old_volume"
```

Comment:
We are tracking similar issue on ResNet: https://github.com/tenstorrent/tt-mlir/issues/1574
Can you add this link in this xfail reason. Somethink like: "Tracking similar issue on ResNet https://github.com/tenstorrent/tt-mlir/issues/1574"